tests/c_languages/CPPCheckBearTest.py

```diff
@@ -1,9 +1,10 @@
 import os
+import unittest
+
```

Comment:
Hm now there's an unnecessary newline at this place 😅 probably GitHub didn't recognize it properly ;)